### PR TITLE
Feature/#2

### PR DESCRIPTION
**Pull request by ChauPham90**
I fixed the issue, please check and let me know if you have any idea

### CODE REVIEW

**Review by RawaaM6 — APPROVED** [review]
i think it looks fine

File: vidly-chau/models/add-gitignore.code-workspace [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (BigBeoU) [inline vidly-chau/models/add-gitignore.code-workspace:1]: why do we have this file?

File: vidly-chau/models/user.js [hunk 2]
```diff
@@ -21,6 +23,10 @@ const userSchema = new mongoose.Schema({
   },
 });
 
+userSchema.methods.generateAuthToken = function () {
```
Comment (BigBeoU) [inline vidly-chau/models/user.js:26]: nice solution :), I was thinking to add seperate function in another place :)

**Review by BigBeoU — APPROVED** [review]
I will merge this and delete other branches

**Pull request merged**